Author: Daniel Baumann <daniel.baumann@progress-technologies.net>
Description: Disabling diag/geodsp temporariily, it currently fails to build.

diff -Naurp syslinux.orig//diag/Makefile syslinux/diag/Makefile
--- syslinux.orig//diag/Makefile	2011-05-16 13:55:10.394573930 +0200
+++ syslinux/diag/Makefile	2011-05-16 13:56:29.094964013 +0200
@@ -1,4 +1,4 @@
-SUBDIRS = mbr geodsp
+SUBDIRS = mbr
 
 all tidy dist clean spotless install:
 	set -e; for d in $(SUBDIRS); do $(MAKE) -C $$d $@; done
